data/templates/SystemInvokerTest.csx

```diff
@@ -0,0 +1,19 @@
+﻿private static string GetQuery(OperationContext<App> cxt)
```

Comment:
OperationContext<App> will not work for ASE,APIM etc, since in tthe Base Class, you are creating the Operation Context based on TResource.

Reply:
try OperationContext<IResource>

Reply:
Great catch:) Changed into IResource.